plugins/fileservices/fileservices.cpp

```diff
@@ -3128,3 +3128,49 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+bool getDefaultValue(const char * processName, const char * propertyName, StringBuffer & strDefaultValue)
+{
+    Owned<IConstEnvironment> daliEnv = openDaliEnvironment();
+    Owned<IPropertyTree> env = getEnvironmentTree(daliEnv);
+
+    if (env.get())
+    {
+        Owned<IPropertyTreeIterator> processIter = env->getElements(processName);
```

Comment:
If these are global defaults, does it make sense to use the values associated with the 1st dfu server?

What is the corresponding implementation for containerized mode?  

(I'm not sure I have a good answer to either question.  @jakesmith ideas?)

Reply:
My original thought/implementation was that, but then it raised several questions/implications, therefore I moved them from Instance to Server Process.

Reply:
Changed